src/inverseRobot.js

```diff
@@ -7,7 +7,17 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const newRobot = {};
+
+  for (const key in robot) {
+    if (newRobot[robot[key]] !== undefined) {
+      return null;
+    }
+
+    newRobot[robot[key]] = key;
```

Comment:
Use object destructuring to improve readability when assigning values from 'robot'.